nautilus_core/model/src/events/order/filled.rs

```diff
@@ -78,3 +141,66 @@ impl Default for OrderFilled {
         }
     }
 }
+
+impl Display for OrderFilled {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(
+            f,
+            "OrderFilled(\
+            instrument_id={}, \
+            client_order_id={}, \
+            venue_order_id={}, \
+            account_id={}, \
+            trade_id={}, \
+            position_id={}, \
+            order_side={}, \
+            order_type={}, \
+            last_qty={}, \
+            last_px={}, \
+            commission={} ,\
+            liquidity_side={}, \
+            ts_event={})",
+            self.instrument_id,
+            self.client_order_id,
+            self.venue_order_id,
+            self.account_id,
+            self.trade_id,
+            self.position_id.unwrap_or_default(),
+            self.order_side,
+            self.order_type,
+            self.last_qty,
+            self.last_px,
+            self.commission
+                .unwrap_or(Money::from_str("0.0 USD").unwrap()),
```

Comment:
If we have what _should_ be an infallible initialization, I think it's cleaner to use `Money::from("0.0 USD")`.

Reply:
Actually, I also had to add this, which we have for `Price` and `Quantity`:

```rust
impl From<&str> for Money {
    fn from(input: &str) -> Self {
        Self::from_str(input).unwrap()
    }
}
```